packages/design-system-tokens/src/lib/types.ts

```diff
@@ -43,7 +43,7 @@ export interface ThemeTokens {
   animation: AnimationTokens;
   color: ColorTokens;
   font: FontTokens;
-  global?: AnyTokenValues;
+  global: AnyTokenValues;
```

Comment:
This change was needed so I could reference `t.global[]` tokens from theme tokens